### ISSUE TITLE
Add CLI documentation

### ISSUE THREAD

**Issue by andyl** [bugmark/bmx_cl_ruby#33]
Add CLI documentation.  Repo is `bugmark/bmx_cl_ruby`.

**Comment by GeorgLink** [comment]
Can you give more information on how one can help?

**Comment by andyl** [comment]
Look in the (command directory)[https://github.com/bugmark/bmx_cl_ruby/tree/master/lib/cmd] and you will find all of our command classes.  For each command there are places to add `desc`, `long_desc` and `option` documentation.  You can look at existing documentation for usage examples, and there is more code examples on the Thor website. http://whatisthor.com/  (we use Thor as our command runner)

ALSO: the README file could use an overview writeup...

http://whatisthor.com/

**Comment by GeorgLink** [comment]
Thanks @andyl

>

**Comment by hritvi** [comment]
hey @andyl! can i work on this?

**Comment by andyl** [comment]
@hritvi yes - plz coordinate w/Georg so that one person is working on this at a time.

**Comment by GeorgLink** [comment]
@hritvi go right ahead. I suggest we push our changes regularly to our fork so that I can see what each other are working on. Additionally, small pull requests with focused improvements will help coordinate our efforts.
